distributed-tests/rackspace-server-manager.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+'''
+A script to create and delete the number of servers on Rackspace
+'''
+
+import pyrax
+import time
+import re
+import os
+import argparse
+import uuid
+
+
+def create_node(nova, counts):
+    flavor = nova.flavors.find(name='2 GB General Purpose v1')
+    image = nova.images.find(name='centos7-test')
+    pubkey = open('key.pub', 'r').read()
+    nova.keypairs.create('distkey', pubkey)
```

Comment:
This may be problematic if we run them in parallel. Can you take a parameter for them? We can use the job name and job number to generate one from Jenkins.